src/main/java/picard/sam/AddOrReplaceReadGroups.java

```diff
@@ -124,6 +130,8 @@ protected int doWork() {
         if (RGPI != null) rg.setPredictedMedianInsertSize(RGPI);
         if (RGPG != null) rg.setProgramGroup(RGPG);
         if (RGPM != null) rg.setPlatformModel(RGPM);
+        if (RGKS != null) rg.setKeySequence(RGKS);
+        if (RGFO != null) rg.setFlowOrder(RGFO);
 
         log.info(String.format("Created read group ID=%s PL=%s LB=%s SM=%s%n", rg.getId(), rg.getPlatform(), rg.getLibrary(), rg.getSample()));
```

Comment:
modify the log to reflect your changes.

Reply:
but be cognizant of possible null.

Reply:
hmmm. on second thought there are a lot of other keys in the RG that we are not printing...perhaps leave as is.

There are no tests, so I can really ask that you modify a test...but FYI, a test would make a wonderful (next) PR!!